.env

```diff
@@ -0,0 +1,3 @@
+#REACT_APP_BASE_URI=http://164.90.247.198
+REACT_APP_BASE_URI=http://localhost:8080
+REACT_APP_CHAIN_ID=1
```

Comment:
This file should not be in source control - the pattern is to commit a `sample.env` file to git, and then users use that to create their own local `.env`. In fact, `.env` should probably be added to `.gitignore`. (A quick google search yields [this article](https://stackabuse.com/managing-environment-variables-in-node-js-with-dotenv/), which explains this [search for "sample"])